config/settings_schema.json

```
@@ -826,67 +826,73 @@
         "default": "1600",
         "label": "t:settings_schema.layout.settings.page_width.label"
       },
+      {
+        "type": "header",
+        "content": "Desktop"
+      },
       {
         "type": "range",
-        "id": "sections_spacing",
+        "id": "spacing_sections_desktop",
         "min": 0,
         "max": 120,
         "step": 4,
         "unit": "px",
         "label": "Vertical spacing between sections",
-        "default": 52
-      },
-      {
-        "type": "paragraph",
-        "content": "Affects collection, search and blog pages. Also affects collage, collection, collection list, multicolumn and blog sections."
-      },
-      {
-        "type": "header",
-        "content": "Desktop grid"
+        "default": 72
       },
       {
         "type": "range",
-        "id": "grid_desktop_horizontal_spacing",
+        "id": "spacing_grid_horizontal_desktop",
         "min": 0,
```

```suggestion
        "min": 4,
```
We won't allow `0` until we plan more time to fix adjacent borders, etc.